crates/ruff_linter/src/rules/pycodestyle/snapshots/ruff_linter__rules__pycodestyle__tests__preview__W391_W391.ipynb.snap

```diff
@@ -0,0 +1,74 @@
+---
+source: crates/ruff_linter/src/rules/pycodestyle/mod.rs
+---
+W391.ipynb:5:1: W391 [*] Too many newlines at end of file
```

Comment:
I think we should change the message to: Too many newlines at the end of the cell